project.clj

```diff
@@ -70,29 +70,31 @@
 
   :dependencies [[org.clojure/clojure ~clj-version]
                  [clj-time "0.15.2"]
-                 [org.threeten/threeten-extra "1.2"]
-                 [clojure.java-time "1.1.0"]
-                 [org.clojure/core.async "1.5.648"]
-                 [org.clojure/core.memoize "1.0.257"]
-                 [org.clojure/tools.logging "1.2.4"]
-                 [org.clojure/tools.cli "1.0.194"]
-                 [pandect "0.6.1"]
-                 [org.clojure/math.combinatorics "0.1.6"]
-                 [version-clj "2.0.1"]
+                 [org.threeten/threeten-extra "1.8.0"]
+                 [clojure.java-time "1.4.3"]
+                 [org.clojure/core.async "1.8.741"]
+                 [org.clojure/core.memoize "1.1.266"]
+                 [org.clojure/tools.logging "1.3.0"]
+                 [org.clojure/tools.cli "1.2.245"]
+                 [pandect "1.0.2"]
+                 [org.clojure/math.combinatorics "0.3.0"]
+                 [version-clj "2.0.3"]
+                 [org.tukaani/xz "1.9"]
+                 [potemkin "0.4.7"]
```

Comment:
Why did you add these 2 ones?
```suggestion
```

Reply:
shall we update this [comment](https://github.com/threatgrid/ctia/blob/748fe1a134647471d89fa7675b3af0642f563455/project.clj#L40) ?